pkgs/tools/misc/cetch/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, writeText, conf ? null }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "cetch";
+  version = "v1.0";
+
+  src = fetchFromGitHub {
+    owner = "wooosh";
+    repo = name;
+    rev = version;
+    sha256 = "1mzlprkyd6imn5lm7srw78zxd1wy9xdb7a2diwpdfx1gfif0iimi";
+  };
+
+  configFile = optionalString (conf!=null) (writeText "config.def.h" conf);
+  preBuild = optionalString (conf!=null) "cp ${configFile} config.def.h";
+
+  makeFlags = [ "DESTDIR=$(out)" ];
+
+  meta = with stdenv.lib; {
+    description = "A fast, highly customizable system info script written in C";
+    homepage = https://github.com/trvv/cetch;
```

Comment:
Hmmm, the homepage and the `fetchFromGitHub` owner do not match.

I get that your fork looks more up-to-date than `trvv`'s, but I am unsure as if yours should be included in NixOS/nixpkgs instead of the original.

Better wait for a maintainer's opinion.